src/components/WeeklyForecast.svelte

```diff
@@ -24,8 +24,7 @@
   const detailsRoute = '/details';
 
   function formatDate(dateString) {
-    return DateTime.fromISO(dateString)
-      .toFormat('cccc dd LLL');
+    return DateTime.fromISO(dateString).toFormat('cccc dd LLL');
```

Comment:
If this is being used only here - it is fine, but don't we use date formatting in other places too? In this case it should be part of some util  module to be accessible globally